azure-pipelines.yml

```diff
@@ -194,6 +194,11 @@ stages:
       parameters:
         testName: 'windows-netframework'
         workingDirectory: 'integrationtest/TargetProjects/NetFramework/FullFrameworkApp.Test'
+    - template: pipeline-templates/run-integration-test-steps.yml
+      parameters:
+        testName: 'windows-initcommand'
+        workingDirectory: 'integrationtest/TargetProjects'
+        strykerCommands: init --config-file "InitCommand/test-config.json" -p "TestProject.csproj"
```

Comment:
I'd like to see 2 tests, 1 with default values, and one that overrides all possible values